### ISSUE TITLE
Popover Controller not working with thingworx 9.2.3

### ISSUE THREAD

**Issue by enthusiasc** [BogdanMihaiciuc/BMPresentationController#21]
Popover controller is not working with thingworx 9.2.3.
Application log has following errors:
```
ERROR [widget] An error occurred while calling BMPopoverController::afterLoad ["BMPopoverController-3",{}]
ERROR [widget] An error occurred while calling destroy for BMPopoverController, Id = "BMPopoverController-3". [{}]
```
See attached sample mashup for reproducing the issue
[Mashups_Test (2).zip](https://github.com/BogdanMihaiciuc/BMPresentationController/files/7482459/Mashups_Test.2.zip)

The pop up window will not open the mashup when calling service bringToFront.
Tested with BMCoreUIWidgets version 2.7 and 2.8.

**Comment by BogdanMihaiciuc** [comment]
Hello,

I'm not able to reproduce the issue as you're describing it here. There's a separate CSS issue that causes the popover to appear blank when using the new 9.0 layouts, but nevertheless the popover does open.

Can you let me know what browser you are testing with?

Thanks.

**Comment by enthusiasc** [comment]
Hey,
this was tested on Chrome Version 92.0.4515.107 (32-Bit) on windows 10.
Everything worked fine with thingworx 9.0. The issue occured after upgrading to thingworx 9.2.3.

**Comment by BogdanMihaiciuc** [comment]
Hi, 

I just tested on 9.2.3 and was able to reproduce this.

That warning is something you can ignore for now, although I will fix it in the next release. The issue is that the new widgets don't emit standard events like `MouseEvent` or `TouchEvent` but instead are instances of `CustomEvent` which don't contain the same information that regular events do.

If your anchor type is set to `Event Origin`, that won't work with any of the new widgets because the events don't contain information about the pointer's position so the popover doesn't know where to display itself.

If your anchor type is set to `Event Target`, this currently fails a type check because of the custom event type and the popover again doesn't know where to appear. This is something that I will be able to fix in the next release.

## TL;DR 
Workaround using either:
 - Set `anchorKind` to `Selector` or `Widget` 
 - Use a legacy button to trigger your popover

**Comment by enthusiasc** [comment]
Hi,
thanks for investigating the issue. Using anchorKind selector is working for me.
